### PR TITLE
MISUV-3931 improve scoverage for CurrencyFormatter

### PR DESCRIPTION
**Pull request by sarinajsal**

### CODE REVIEW

**Review by andriikondiuk — APPROVED** [review]
Looks good to me

File: test/testUtils/CurrencyFormatterSpec.scala [hunk 1]
```diff
@@ -36,6 +36,23 @@ class CurrencyFormatterSpec extends TestSupport with ImplicitCurrencyFormatter {
       }
     }
 
+    "CurrencyFormatterInt.toNegativeCurrency" should {
+      "return a value as negative" when {
+        "the number is more than 0" in {
+          val amount: Int = 1
```
Comment (avelx) [inline test/testUtils/CurrencyFormatterSpec.scala:42]: * not sure why we use Int to model amount. maybe BigDecimal?, minor thing;
* to be 100% sure I would use property base testing to check if this conversion work for all variation of inputs(this is optional);

**Review by avelx — APPROVED** [review]
minor comment, looks good otherwise

File: test/testUtils/CurrencyFormatterSpec.scala (hunk 1 above)
Comment (sarinajsal) [reply test/testUtils/CurrencyFormatterSpec.scala:59]: We had two implicits for bigDecimal and Int, we've added tests for the Int and now have just pushed for the BigDecimal as well. You're right, property based tests are the best but for this case its should be sufficient (Andrii) Thank you!

Comment (sarinajsal) [reply test/testUtils/CurrencyFormatterSpec.scala:59]: Pushed big decimal tests, is it ok if Andrew merges?

Comment (avelx) [reply test/testUtils/CurrencyFormatterSpec.scala:59]: yeh, sure

**Review by avelx — APPROVED** [review]

**Pull request merged**